qt/logviewdialog.py

```diff
@@ -178,49 +178,15 @@ def comboFilterChanged(self, index):
 
     def contextMenuClicked(self, point):
         menu = QMenu()
-        clipboard = qttools.createQApplication().clipboard()
         cursor = self.txtLogView.textCursor()
 
-        btnCopy = menu.addAction(_('Copy'))
-        btnCopy.triggered.connect(lambda: clipboard.setText(cursor.selectedText()))
-        btnCopy.setEnabled(cursor.hasSelection())
-
-        btnAddExclude = menu.addAction(_('Add to Exclude'))
-        btnAddExclude.triggered.connect(self.btnAddExcludeClicked)
-        btnAddExclude.setEnabled(cursor.hasSelection())
-
         btnDecode = menu.addAction(_('Decode'))
         btnDecode.triggered.connect(self.btnDecodeClicked)
         btnDecode.setEnabled(cursor.hasSelection())
         btnDecode.setVisible(self.config.snapshotsMode() == 'ssh_encfs')
```

Comment:
It seems to me that only the entry "Decode" is left in this context menu. This is visible only when the current snapshot profile use encryption (EncFS).

So I propose to add the check `self.config.snapshotsMode() == 'ssh_encfs'` to line 147 where the context menu trigger is installed. It should be installed only when there is a encrypted (encfs) snapshot profile.

Side note: What confuses me is that the check is only for `ssh_encfs` and not also `local_encfs`. Maybe you can check this out if there is a good reason for this? But be aware that we plan to remove (replace) encfs in the long run (#1549). So don't invest to much resources into this encfs-thing.

Reply:
Hi. I worked with T1petitti in a group of 3. The Decode option does not come up graphically because the context menu does not appear either, which is why we didn't think we needed to change anything involving the Decode feature, but moving the condition to before the context menu is created instead of after is something we can do. 

https://github.com/bit-team/backintime/commit/3bc25fa3f5fd135e4eada8cdbbcd9e2b1079c06f
This commit by @Germar is where it got added. It does not say why it got added. 
Is it possible 'local_encfs' does not have a decode option because local file-paths, not through ssh, do not need to be decoded?

Reply:
Great! :rocket: A team of three. :love_you_gesture: 

I don't use EncFS and so I don't know the details. Might it be possible that you can give it a short test: Create one "locale (encrypted)" and "SSH (encrypted)" profile and then check this feature?